src/Capability.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Capability interface declaring the known capabilities.
+ *
+ * @package Requests
+ * @subpackage Capability
```

Comment:
Related to #590 - should this be changed ? Do we want to add a new subcategory or does this belong in `Utilities` ?

```suggestion
 * @package Requests\Utilities
```